README.md

```diff
@@ -42,82 +42,92 @@ Add view for your layout:
 <io.github.kbiakov.codeview.CodeView
 	android:id="@+id/code_view"
 	android:layout_width="wrap_content"
-	android:layout_height="wrap_content"/>
+	android:layout_height="wrap_content"
+	app:animateOnStart="true" />
 ```
 
 Use chaining syntax when build view:
 ```java
 CodeView codeView = (CodeView) findViewById(R.id.code_view);
 
-codeView.highlightCode("js")
-        .setColorTheme(ColorTheme.SOLARIZED_LIGHT.withBgContent(myColor))
-        .setCodeContent(getString(R.string.listing_js));
+new Highlighter(this)
```

Comment:
Also there is no need to create new ```Highlighter``` instance when you want just to replace code.